.gitignore

```diff
@@ -47,6 +47,7 @@ tsconfig.tsbuildinfo
 /contracts/deployments/river_anvil
 /contracts/deployments/alpha/base_anvil
 /contracts/deployments/alpha/river_anvil
+/contracts/deployments/local_multi
```

Comment:
there are other places too, but you'll definitely want it in the git ignore. the two scripts that get called locally are 

```
./scripts/configure-nodes.sh --multi
```
 and
```
./scripts/configure-nodes.sh --multi_ne
```

which means its `local_multi_ne` not de! sorry!

Reply:
fixed in latest push, will take a context and remove that folder